gf-ui-lib/src/components/Title/Title.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./Title.module.scss";
+
+/**
+ * Title props
+ */
+
+interface TitleProps {
+  /**
+   * Text of the title
+   */
+  text: string;
+
+  /**
+   * Title's size
+   */
+  size: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
WHat do you think about create a enum with title levels?

Reply:
I think that enum is redundant here. We won't iterate over heading levels.

Reply:
But it is very similar to the classic enum with the seasons: winter, spring, summer, autumn. And if we will add other tags (for example, p or div or other) then enum better then long list.

Reply:
Enums are not for iterating. I don't understand metaphor with seasons. :) Also it looks like not size, but level. I vote for enum because it more redundant. Even without enum I propose to create separate type.